etc/ci/upload_nightly.sh

```diff
@@ -16,8 +16,8 @@ upload() {
     package_filename="$(basename "${2}")"
     local -r nightly_upload_dir="s3://servo-builds/nightly/${1}"
     local -r package_upload_path="${nightly_upload_dir}/${package_filename}"
-    s3cmd put "${2}" "${package_upload_path}"
-    s3cmd cp "${package_upload_path}" "${nightly_upload_dir}/servo-latest.${3}"
+    s3cmd --mime-type="application/octet-stream" put "${2}" "${package_upload_path}"
```

Comment:
This goes beyond 80 characters on the line; I would suggest breaking right before "put".